local.yml

```diff
@@ -0,0 +1,11 @@
+- hosts: localhost
```

Comment:
This looks like a test playbook I assume.

Reply:
yes, since I couldn't test this work at all wrote some dummy playbook to test some of the logic.